pkgs/applications/science/biology/bowtie2/default.nix

```diff
@@ -1,26 +1,26 @@
-{ stdenv, fetchFromGitHub, zlib, tbb, python, perl }:
+{ stdenv, fetchFromGitHub, zlib, tbb, python3, perl }:
 
 stdenv.mkDerivation rec {
   pname = "bowtie2";
-  version = "2.3.5.1";
+  version = "2.4.2";
 
   src = fetchFromGitHub {
     owner = "BenLangmead";
     repo = pname;
     rev = "v${version}";
-    sha256 = "1l1f0yhjqqvy4lpxfml1xwv7ayimwbpzazvp0281gb4jb5f5mr1a";
+    sha256 = "11apzq7l1lk3yxw97g9dfr0gwnvfh58x6apifcblgd66gbip3y1y";
   };
 
-  buildInputs = [ zlib tbb python perl ];
+  buildInputs = [ zlib tbb python3 perl ];
```

Comment:
I think python3 could be a nativeBuilInputs. Not sure about perl.

Maybe we should think about using CMake because the project supports it.

Reply:
cmake support added in 9139eb5aeeb11cd3bce46f8c8a4984698331b56e